src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -5975,16 +5975,19 @@ class C
     {
         switch (true)
         {
-            case identifier $$
+            case identifier$$
         }
     }
 }
                               </Document>,
                   showCompletionInArgumentLists:=showCompletionInArgumentLists)
 
-                state.SendTypeChars("z")
+                state.SendTypeChars(" ")
```

Comment:
i would perfer not changing what the test *does* only what it checks.  if we want to test a different set of user behaviors, we shoudl add a new test.

Reply:
I agree with this in general, but in this particular case, I'd argue that the changed behavior still reflects the original intention of the test (based on the test name at least)

Reply:
the test name can be updated to now reflect what it was previously doing.  you can then use thsi test name to create a new test that does something different.  Tests are good, even if they're testing something else, and i'd prefer we never get rid of an existing test unless there is a very very good reason to do so (e.g. the component they test doesn't even exist anymore).

Reply:
OK. In that case, there's probably no point to cove the new behavior here, which is covered in a separate test. Reverted this to its original behavior.